c2cgeoportal/scaffolds/create/+dot+travis.yml

```diff
@@ -0,0 +1,27 @@
+language: python
+
+python:
+- 2.7
+
+install:
+- sudo apt-get install postgis
+- sudo -u postgres createdb -E UTF8 -T template0 template_postgis
+- sudo -u postgres psql -d template_postgis -c "CREATE EXTENSION postgis;"
+- sudo -u postgres createdb -E UTF8 -T template_postgis geomapfish
```

Comment:
When we just create the project it's the case :-)
https://github.com/camptocamp/c2cgeoportal/blob/master/c2cgeoportal/scaffolds/create/vars_%2Bpackage%2B.yaml_tmpl#L5